test/interchaintest/interchainstaking_test.go

```diff
@@ -116,120 +116,120 @@ func TestInterchainStaking(t *testing.T) {
 	_ = quickUserAddr
 	_ = junoUserAddr
 
-	runSidecars(t, ctx, quicksilver, juno)
+	// runSidecars(t, ctx, quicksilver, juno)
 }
 
-func runSidecars(t *testing.T, ctx context.Context, quicksilver, juno *cosmos.CosmosChain) {
-	t.Helper()
-
-	runICQ(t, ctx, quicksilver, juno)
-	// runXCC(t, ctx, quicksilver, juno)
-}
-
-func runICQ(t *testing.T, ctx context.Context, quicksilver, juno *cosmos.CosmosChain) {
-	t.Helper()
-
-	var icq *cosmos.SidecarProcess
-	for _, sidecar := range quicksilver.Sidecars {
-		if sidecar.ProcessName == "icq" {
-			icq = sidecar
-		}
-	}
-	require.NotNil(t, icq)
-
-	containerCfg := "config.yaml"
-
-	file := fmt.Sprintf(`default_chain: '%s'
-chains:
-  '%s':
-    key: default
-    chain-id: '%s'
-    rpc-addr: '%s'
-    grpc-addr: '%s'
-    account-prefix: quick
-    keyring-backend: test
-    gas-adjustment: 1.2
-    gas-prices: 0.01uqck
-    min-gas-amount: 0
-    key-directory: %s/.icq/keys
-    debug: false
-    timeout: 20s
-    block-timeout: 10s
-    output-format: json
-    sign-mode: direct
-  '%s':
-    key: default
-    chain-id: '%s'
-    rpc-addr: '%s'
-    grpc-addr: '%s'
-    account-prefix: osmo
-    keyring-backend: test
-    gas-adjustment: 1.2
-    gas-prices: 0.01uosmo
-    min-gas-amount: 0
-    key-directory: %s/.icq/keys
-    debug: false
-    timeout: 20s
-    block-timeout: 10s
-    output-format: json
-    sign-mode: direct
-`,
-		quicksilver.Config().ChainID,
-		quicksilver.Config().ChainID,
-		quicksilver.Config().ChainID,
-		quicksilver.GetRPCAddress(),
-		quicksilver.GetGRPCAddress(),
-		icq.HomeDir(),
-		juno.Config().ChainID,
-		juno.Config().ChainID,
-		juno.GetRPCAddress(),
-		juno.GetGRPCAddress(),
-		icq.HomeDir(),
-	)
-
-	err := icq.WriteFile(ctx, []byte(file), containerCfg)
-	require.NoError(t, err)
-	_, err = icq.ReadFile(ctx, containerCfg)
-	require.NoError(t, err)
-
-	err = icq.StartContainer(ctx)
-	require.NoError(t, err)
-
-	err = icq.Running(ctx)
-	require.NoError(t, err)
-}
-
-func runXCC(t *testing.T, ctx context.Context, quicksilver, juno *cosmos.CosmosChain) {
-	t.Helper()
-
-	var xcc *cosmos.SidecarProcess
-	for _, sidecar := range quicksilver.Sidecars {
-		if sidecar.ProcessName == "xcc" {
-			xcc = sidecar
-		}
-	}
-	require.NotNil(t, xcc)
-
-	containerCfg := "config.yaml"
-
-	file := fmt.Sprintf(`source_chain: '%s'
-chains:
-  quick-1: '%s'
-  juno-1: '%s'
-`,
-		quicksilver.Config().ChainID,
-		quicksilver.GetRPCAddress(),
-		juno.GetRPCAddress(),
-	)
-
-	err := xcc.WriteFile(ctx, []byte(file), containerCfg)
-	require.NoError(t, err)
-	_, err = xcc.ReadFile(ctx, containerCfg)
-	require.NoError(t, err)
-
-	err = xcc.StartContainer(ctx)
-	require.NoError(t, err)
-
-	err = xcc.Running(ctx)
-	require.NoError(t, err)
-}
+// func runSidecars(t *testing.T, ctx context.Context, quicksilver, juno *cosmos.CosmosChain) {
+// 	t.Helper()
+
+// 	runICQ(t, ctx, quicksilver, juno)
+// 	// runXCC(t, ctx, quicksilver, juno)
+// }
```

Comment:
can you add the sidecar setup back in now that `interchaintest` supports it?

Reply:
Yess!